### DISCUSSION TITLE
第一次运行能否提示下要生成默认配置文件?

### DISCUSSION THREAD

**Discussion by dfaofeng** [yuudi/gypsum#5] [想法 Ideas]
不应该一点就闪屏,还以为下载的程序有问题了

**Comment by yuudi** [comment]
应该是有的，，这个版本有 bug 所以没显示出来。。。

**Comment by yuudi** [comment]
已修复 yuudi@d799b53fa0fe8d7aef5f624f35b3c46eb5e379d6

**Reply by dfaofeng** [reply]
还有这个后端项目的删除任务功能是不是没写.api直接报404了  http://127.0.0.1:9900/api/v1/schedulers/2
![image](https://user-images.githubusercontent.com/14891398/110091592-936e8f80-7dd3-11eb-8056-c6c26ff89f2a.png)

**Reply by yuudi** [reply]
。。。是 bug，下个版本修
